clients/rancher/generated/management/v3/zz_generated_key_cloak_oidcconfig.go

```diff
@@ -3,6 +3,7 @@ package client
 const (
 	KeyCloakOIDCConfigType                     = "keyCloakOIDCConfig"
 	KeyCloakOIDCConfigFieldAccessMode          = "accessMode"
+	KeyCloakOIDCConfigFieldAcrValue            = "acrValue"
```

Comment:
Why weren't these changes in the initial PR?

Reply:
actually when i started working on 1.30 i updated shepherd with my fork. After that some PRs got merged in rancher and i rebased my rancher PR but I did not updated my fork which was being used in shepherd.
So a few PRs regarding the above changed fields went in rancher that's why these changes are now occurring when i am updating rancher pkg/apis to the latest.

Reply:
@caliskanugur i did go generate in earlier PR as well since wrangler was updated from v2 to v3 that was needed.

Reply:
Sounds good to me, thanks @vardhaman22